mergify_engine/hooks.py

```diff
@@ -0,0 +1,81 @@
+# -*- encoding: utf-8 -*-
```

Comment:
I'd rather call that `signal` than `hooks` since it's mostly a one way message being sent.